starter/src/main/java/com/example/starter/base/osgi/FixedVaadinServlet.java

```diff
@@ -9,16 +9,7 @@
 
 import com.vaadin.flow.server.VaadinServlet;
 
-@Component(service = Servlet.class
```

Comment:
This change should be reverted with changing compat mode to production mode.
`@VaadinMode`  allows to replace long property line in the `@Component`.
This is said here: `Same like @VaadinMode`.

`@VaadinMode` allow to specify `productionMode` property value.
It's `false` by default but this is not production code which should be short.
This is an example which demoes how to do something.
So it contains "placeholder" for the developer to extend the example with this own code.
Here the developer should understand:
- he can specify _ANY_ Flow init parameter directly via `property`  (with the prefix)
- he can do the same in a Java way via extending `@VaadinMode` annotation.

Reply:
Got it now, thanks a lot! These are two separate ways to set the init parameters. The second one is type-safe and can be hidden to a custom annotation.